lib/cudnn/nnlib.jl

```diff
@@ -67,6 +94,7 @@ fix1d(cdims::DenseConvDims{1,K,C_in,C_out,S,P,D,F}) where {K,C_in,C_out,S,P,D,F}
 fix1d(pdims::PoolDims{1,K,S,P,D}) where {K,S,P,D,F} =
   PoolDims{2,(K...,1),(S...,1),(P...,0,0),(D...,1)}((pdims.I..., 1), pdims.C_in)
 
+conv_forward = CircularDict{Tuple, Int32}(100)
```

Comment:
Why only cache a limited amount? If you key with not much uniqueness, only `(T, size, DenseConvDims)`, we should have a high enough hit rate? The entries are small, too, and don't keep anything alive.

Reply:
Cf. my comment here: https://github.com/JuliaGPU/CUDA.jl/pull/321#issuecomment-665542278